test/swap/Ord_Marketplace.t.sol

```diff
@@ -305,6 +305,24 @@ contract OrdMarketPlaceTest is OrdMarketplace, Test {
         assertEq(ids.length, 0);
     }
 
+    function test_acceptOrdinalSellOrderWhenOrderIsAlreadyWithdrawnShouldRevert() public {
+        token1.sudoMint(bob, 100);
+
+        // placeOrdinalSellOrder by alice
+        setUpForAcceptOrdinalSellOrder();
+
+        // withdraw sell order
+        this.withdrawOrdinalSellOrder(0);
+
+        // try accepting withdraw order
+        vm.startPrank(bob);
+        token1.approve(address(this), 100);
+        // since order is withdrawn it will try to send tokens to default address
+        vm.expectRevert("Address: call to non-contract");
+        this.acceptOrdinalSellOrder(0, ordinalsInfo[0].requester);
```

Comment:
ideally, it should check if the default value is returned from storage and then throw a custom error such as "Order not present". 
But this then does require additional checks in the ordinal contract not sure if we want to add that, as business logic is not getting affected.